simpleLooping/test_isPrime.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from isPrime import isPrime
+
+class TestIsPrime(unittest.TestCase):
```

Comment:
You don't need to use big number in your unit test as it will slow down the test, only use it to test small number. Remember, unit test is to verify if your code work correctly, not to test the performance (aka stress test),